projects/ngx-sub-form/src/lib/ngx-root-form.component.spec.ts

```diff
@@ -52,7 +52,11 @@ class TestWrapperComponent {
       <input type="text" data-color [formControlName]="formControlNames.color" />
       <input type="radio" data-can-fire [formControlName]="formControlNames.canFire" value="true" />
       <input type="radio" data-can-fire [formControlName]="formControlNames.canFire" value="false" />
-      <input type="number" data-number-of-people-on-board [formControlName]="formControlNames.numberOfPeopleOnBoard" />
+      <input
+        type="number"
+        data-number-of-crew-members-on-board
```

Comment:
`data-number-of-crew-members-on-board` --> `data-nb-crew-members`?

Reply:
as above `data-crew-member-count`